### ISSUE TITLE
Create a spreadsheet cross-referencing Chromium HTML feature counters with Servo implementation status

### ISSUE THREAD

**Issue by pcwalton** [servo/servo#6905]
Chromium now has this page: https://www.chromestatus.com/metrics/feature/popularity

It would be great to create a spreadsheet cross-referencing this chart with Servo implementation status so we can understand which HTML and DOM features are most important to implement. We did this with CSS and IMHO it was very successful.

**Comment by pcwalton** [comment]
One problem with this, though, is that it looks like Chromium is doing this on a case-by-case basis in order to decide whether to remove or break stuff, so overall coverage seems quite low in comparison to the CSS feature list.

**Comment by s142857** [comment]
the page is not scrolling

**Comment by jdm** [comment]
The CSS spreadsheet is at https://docs.google.com/spreadsheets/d/1CxLS8w8GwK-2euVErrqpUUb76PiZa6w5h5EnGsL9KFs .

**Comment by jdramani** [comment]
@pcwalton @jdm Hi, I would like to do this if it's okay. I am new to open source and finding ways to contribute to servo. What else would i need to make the spreadsheet other than https://www.chromestatus.com/metrics/feature/popularity ?

**Comment by jdm** [comment]
The other data comes from Servo itself; presumably we would want to mark each feature as "not started", "started", "mostly complete", or "complete". An easy approximation of that would be to look at the appropriate files in `components/script/dom/webidls` and see how many things are currently commented out, since those correspond to missing pieces of that API.

**Comment by metajack** [comment]
Note that creating half the spreadsheet is still useful, and contributors can work on filling in Servo support.

**Comment by LifeIsStrange** [comment]
It already exist :) 
https://html5test.com/

And the github :https://github.com/NielsLeenheer/html5test

**Comment by mruszczyk** [comment]
I've translated the data at the link above into a basic spreadsheet. Unsure of where to begin with regards to servo support on a decent chunk of the properties.

https://docs.google.com/spreadsheets/d/1IssfSeHo2GJf5jRtwkuJJhsl38PW8KQ6cY0AXc0iNps/edit?usp=sharing

**Comment by jdm** [comment]
Yeah, many of these have names which don't clearly map to particular web features, unfortunately.

**Comment by jdm** [comment]
If I get edit access to the document I can try and leave comments about what I think some names mean. Otherwise I can only suggest grepping the chromium source for the names and looking at the file blame for the commit that introduced them, which should provide more context.

**Comment by mruszczyk** [comment]
Should be granted.

**Comment by jdm** [comment]
...so many entries...

**Comment by nox** [comment]
Was this done?

**Comment by KiChjang** [comment]
I don't think I ever saw a spreadsheet or a website that tracks missing HTML/DOM features of servo.
